scripts/globals/items/bottle_of_catholicon.lua

```diff
@@ -0,0 +1,94 @@
+-----------------------------------------
+-- ID: 4206
+-- Item: Catholicon
+-- Item Effect: This elixir is known for its ability to cure most status ailments.
+-- Removes up to three negative side effects.
+-- Does not remove Curse, Doom, or Amnesia.
+-- It can be used on any player, Even those not in your own Party.
+-----------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0
+    if target:hasStatusEffect(dsp.effect.MEDICINE) then
+        result = 111
+    end
+    return result
+end
+
+function onItemUse(target)
+    local removables =
+    {
+        dsp.effect.SILENCE,
+        dsp.effect.BLINDNESS,
+        dsp.effect.POISON,
+        dsp.effect.PARALYSIS,
+        dsp.effect.DISEASE,
+        dsp.effect.PETRIFICATION,
+        dsp.effect.BIND,
+        dsp.effect.WEIGHT,
+        dsp.effect.SLOW,
+        dsp.effect.GRADUAL_PETRIFICATION,
+        dsp.effect.ADDLE,
+        dsp.effect.MUTE,
+        dsp.effect.BANE,
+        dsp.effect.PLAGUE,
+        dsp.effect.BURN,
+        dsp.effect.FROST,
+        dsp.effect.CHOKE,
+        dsp.effect.RASP,
+        dsp.effect.SHOCK,
+        dsp.effect.DROWN,
+        dsp.effect.DIA,
```

Comment:
most of this can be eliminated by checking for flag erasable

Reply:
yeah thats what i thought but it turns out some of the things wont remove from just the flag, like poison for some reason, maybe flag is wrong?, im stupid? who knows lol

Reply:
I can copy what is in the eraser attachment script, that checks for erasable flag but also uses a table of a few non eraseable statuses to iterate through

Reply:
> yeah thats what i thought but it turns out some of the things wont remove from just the flag, like poison for some reason, maybe flag is wrong?,

poison isn't erasable, thats why I said "most"

> I can copy what is in the eraser attachment script, that checks for erasable flag but also uses a table of a few non eraseable statuses to iterate through

sounds like a plan :+1:

Reply:
As suggested, now done.